### ISSUE TITLE
Add Text Wrap feature

### ISSUE THREAD

**Issue by LaurenWags** [brave/browser-android-tabs#320]
**Did you search for similar issues before submitting this one?**
Yes

**Description:**
Requested via twitter/community: https://community.brave.com/t/add-zoomed-text-reflow-wrap-feature-like-opera-and-firefox/5014

Opera (based on Chromium) has a core feature called Text Wrap that, after zooming in on a block of text, will automatically reflow the text at that size to fit the viewport. Firefox has an add-on that does something similar (although you have to tap each paragraph to reflow it). You can see it in action here: https://twitter.com/stowball/status/888580969602531328

It would be great for accessibility if Brave also implemented this feature, as it’s often much easier to read websites at a size larger than the developers determined necessary, and is available everywhere (unlike features like Reader mode).

**Comment by te0006** [comment]
Opera is the ONLY major browser on Android that gets this right. After activating "Text Reflow" in the configuration page it just works, by default and on each and every text-heavy web page, without marking or extra clicks. 
Unfortunately, after the buyout I  really no longer trust Opera but I have to keep the damn thing simply because my eyesight is no longer 20/20. 
I cannot fathom how this issue, ruining the mobile browsing experience for so many of us, is neglected in both FF and Chrome. The sad thing is Google was already so close to implementing reflow in Chrome on Android but then dropped it, just like that, in order to save 2% performance and one configuration switch, and probably also to pursue their illusion that they can force all websites to play nice. Surprise - they don't,  but fortunately that's the users' problem not Google's. 
Implement this feature and use it in marketing. There are many people out there needing it desperately.

**Comment by gerardp** [comment]
Any news on this? text reflow is the only feature for which I still use opera.

**Comment by thousandsofthem** [comment]
Bump

**Comment by Marvelicious75** [comment]
I'm interested in this feature also. Every few months I go on a hunt for mobile browsers again and discover that despite its other flaws, I wind up using opera simply because I can read the text on sites I visit, functionally zoom in on a picture, etc...

**Comment by Javi** [comment]
Between text reflow and having a toolbar on the bottom of the screen (i.e. where hands rest naturally), Opera is the current champion of usability in the whole web browser market. Getting there with an open source browser like brave could be a defining marketing advantage, I strongly suggest you implement those features.

**Comment by aimutu** [comment]
I found a solution, Yandex Browser can do it

**Comment by OrdinaryGeekSF** [comment]
I would LOVE it if Brave could do this! At the moment, I use Opera as my default browser, BECAUSE OF text reflow. PLEASE bring this feature to Brave!

**Comment by JazzTech** [comment]
Please implement this! 

I really want to leave Opera now that they have been bought out by China, but auto-text reflow is a mandatory feature for me - darned elderly eyes ...

**Comment by airportcabbie** [comment]
Currently (Jan 2020) the only 2 android browsers I've found, other than Opera, that do text reflow are "Naked browser" and Firefox via the "Fennec" extension.

**Comment by Marvelicious75** [comment]
Yep, and Opera seems to be the only one that works more than sporadically.
I read somewhere that they use some copyrighted code for that particular
operation. Frustrating.
